src/GRA.Data/Repository/AvatarBundleRepository.cs

```diff
@@ -136,6 +177,19 @@ public async Task<bool> IsItemInBundle(int itemId, bool? unlockable = null)
             return await inBundle.AnyAsync();
         }
 
+        public List<int> GetBundleId(int itemId)
+        {
+            return _context.AvatarBundleItems
+                .AsNoTracking()
+                .Where(_ => _.AvatarItemId == itemId && !_.AvatarBundle.IsDeleted)
+                .Select(_ => _.AvatarBundleId).ToList();
+        }
+
+        public AvatarBundle GetItemsBundles(int bundleIds)
```

Comment:
This method should be made `async`, return `Task<AvatarBundle>` and contain `return await GetByIdAsync(bundleIds);`